helm-org.el

```diff
@@ -396,7 +396,12 @@ will be refiled."
   (let* ((victims (with-helm-buffer (helm-marked-candidates)))
          (buffer (marker-buffer marker))
          (filename (buffer-file-name buffer))
-         (rfloc (list nil filename nil marker)))
+         ;; get the heading we refile to so org doesn't
+         ;; output 'Refile to "nil" in file ...'
+         (heading (with-current-buffer buffer
```

Comment:
`with-current-buffer` should be unnecessary, because that should be covered by `(org-with-point-at- marker...`.